docs/user-manuals/network/edge-pod-network.md

```diff
@@ -64,6 +67,11 @@ git reset --hard 9ebe139e77e82afb122e335328007bca86905ae4;
 wget https://raw.githubusercontent.com/openyurtio/openyurt/master/docs/tutorial/0002-ipam-keep-pod-ip.patch;
 git am 0002-ipam-keep-pod-ip.patch;
 ```
+host-local的cni rpm包部署
+```
+rpm -ivh https://xhzy-software-package.oss-cn-hangzhou.aliyuncs.com/openyurt/openyurt-cni-0.8.7-0.x86_64.rpm
```

Comment:
cni rpm has been uploaded at here: https://github.com/openyurtio/openyurt/releases/download/v0.6.0/openyurt-cni-0.8.7-0.x86_64.rpm
please use this link url.